src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -78,7 +78,7 @@
       <Link>Modules\NuGet\NuGet.Format.ps1xml</Link>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="..\NuGetConsole.Host.PowerShell\Scripts\NuGet.psd1">
+    <Content Include="$(BaseIntermediateOutputPath)$(Configuration)\NuGet.psd1">
```

Comment:
Is this signed correctly after this change?

Reply:
It wasn't, but I fixed it with https://github.com/NuGet/NuGet.Client/pull/2863/commits/b0783a1b58aac3ef1694f3ce3afc0332119a1968.